packages/client/hmi-client/src/components/dataset/tera-dataset.vue

```diff
@@ -53,10 +51,13 @@
 					<section>{{ dataset?.username || '-' }}</section>
 				</section>
 			</section>
-			<section class="metadata data-row" v-if="!metadata">
+			<section class="metadata data-row">
 				<section>
 					<header>Source Name</header>
-					<section>{{ dataset?.source || '-' }}</section>
+					<section v-if="dataset.url === 'https://github.com/reichlab/covid19-forecast-hub/'">
```

Comment:
I'm assuming so, but, to confirm we want to hardcode this github url and title into `tera-dataset.vue`?

Reply:
that should not be here for some resons.